app/assets/stylesheets/common/modal/modal-overrides.scss

```diff
@@ -310,11 +310,7 @@
 .d-modal.fullscreen-table-modal {
   .d-modal {
     &__container {
-      max-width: $reply-area-max-width;
```

Comment:
This prevents the modal being fully 100%

Reply:
it does, but that's how it was originally I think? Based on what was decided here: https://meta.discourse.org/t/table-builder/236016/6?u=chapoi

Reply:
Okay let's leave the table one as is, I don't care about that too much, but can we please just make the code one full screen? The code one just looks terrible ATM, and when I click a fullscreen button I expect it to at least full up _most_ of the screen, not just be a little floating window in the middle.

![image](https://github.com/discourse/discourse/assets/920448/2cb76bfe-2ad0-435e-9b76-6cb3c7246efa)

Reply:
And a better local example...really not a fan of this, may as well not click the button:

![image](https://github.com/discourse/discourse/assets/920448/73d76398-57fa-43c6-8c06-3e98efff7ee7)

I think it needs to get a lot bigger and fill most of the screen.

Reply:
Yeah its weird this doesnt grow. Let me take a look with that specific code example

Reply:
I agree it should grow to where it can, to avoid a horizontal scrollbar. But what would you gain by making if full-width compared to:
<img width="1481" alt="image" src="https://github.com/discourse/discourse/assets/101828855/116ba7d4-e9fd-4214-9a6c-5145f7b08016">

Reply:
As long as it grows and doesn't show the horizontal scroll I am happy 👍